server/metrics/metrics.go

```diff
@@ -20,12 +19,15 @@ var (
 	Counters = make(map[string]*prometheus.CounterVec)
 )
 
-func init() {
+// StartMetrics initializes and handles metrics Prometheus endpoint
+func StartMetrics() {
```

Comment:
Why change this to an explicit call instead of package init? My understanding (not tested) is that you can initialise Prometheus http handler and metrics independently.

Reply:
This was needed because init is ran when the file is accessed or prior to any other function. Due to the way the server is setup a metric endpoint was being set up for each server(login, world, and all channels). During testing there were some apparent race conditions appearing due to the use of init rather than an explicit call. This was primarily that the metric Port variable was not able to get set prior to the initialization of the endpoint. By making this an explicit call rather than using init we are able to control the initialization of the endpoint and prevent race conditions.